app/controllers/job_completion_summary_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class JobCompletionSummaryController < ApplicationController
+  before_action :set_job_completion_summary, only: [:show]
+
+  def index
+    @job_completion_summary = JobCompletionSummary.recent_errors.page(params[:page])
```

Comment:
Can we keep the language consistent? So like JobCompletionSummary.recent_completions